src/panel/direction/DirectionPanel.jsx

```diff
@@ -69,7 +69,10 @@ export default class DirectionPanel extends React.Component {
       const origin = new NavigatorGeolocalisationPoi();
       try {
         await origin.geolocate({ displayErrorModal: false });
-        this.setState({ origin, originInputText: origin.name });
+        this.setState(
+          { origin, originInputText: origin.name },
+          this.computeRoutes
```

Comment:
In other cases we use `this.update`, which also updates the url. It would be better here also for consistence, except if you have a reason to only call `computeRoutes`.

Reply:
ok, so let's call this.update instead